### ISSUE TITLE
Canvas should be used with "caution"

### ISSUE THREAD

**Issue by bentruyman** [h5bp/html5please#106]
As someone with a bit of experience using FlashCanvas spending a number of hours tracing through Actionscript, I'm on the fence as to whether or not canvas should be classified as "use" or "caution". It isn't a drop-in polyfill that's going to transparently produce the kind of results we typically expect from most polyfills. FlashCanvas, in my experience, handles basic shapes and simple animations just fine. But as soon as you take it outside of it's narrow comfort zone, it falls apart.

Also, I should note, the author of the project states that FlashCanvas' does fairly well when put up against this test suite, which it does. However, where FlashCanvas often fails is when you actually throw it into a real-world situation where we're performing a several operations on the canvas at once (like a `globalCompositionOperation` after a `drawImage`).

For example, see the limitations with the `drawImage` method here: https://groups.google.com/group/flashcanvas/browse_thread/thread/a16eea741872b085/d0e66cc44b1441c6

Awhile back, I created a demo to test performance in IE: http://bentruyman.com/lab/canvas/dyo/

I apologize I don't have a more complex example to prove my point. :(

tl;dr Without a better alternative than FlashCanvas, I'd say this one should be classified as "caution".
